commonItems/UnitTests/SystemUtilsTests.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+using System.IO;
+using Xunit;
+
+namespace commonItems.UnitTests {
+    [Collection("Sequential")]
+    [CollectionDefinition("Sequential", DisableParallelization = true)]
+    public class SystemUtilsTests {
+        readonly string testFilesPath = "UnitTests/TestFiles";
+        [Fact] public void GetAllFilesInFolderDoesntWorkRecursively() {
+            var files = SystemUtils.GetAllFilesInFolder(testFilesPath);
+            Assert.Single(files);
```

Comment:
Can you make it more explicit? It'd be nice to have it show the format of the files in the SortedSet. For instance, is the path included in the filename or not? If I were using this library, I could verify that be looking at the test.